self_enrollment_tool/views.py

```diff
@@ -355,55 +356,24 @@ def disable(request, uuid):
     Removes course from self enrollment table.
     Users will no longer be able to self enroll in course.
     """
-    logger.info(f'Deleting self-enroll course uuid:{uuid}.')
+    logger.info(f'Deleting self-enroll URL for course uuid:{uuid}.')
 
     try:
         try:
             self_enrollment_course = SelfEnrollmentCourse.objects.get(uuid=uuid)
         except SelfEnrollmentCourse.DoesNotExist:
-            msg = f'Course (uuid:{uuid}) does not exists and therefore cannot be deleted.'
+            msg = f'Self-enroll URL for course (uuid: {uuid}) does not exists and therefore cannot be deleted.'
             logger.warning(msg)
             messages.warning(request, msg)
 
         self_enrollment_course.delete()
     except Exception:
-        msg = f'Unable to delete self-enroll course uuid:{uuid}.'
-        logger.exception(msg)
-        messages.error(request, msg)
-
-    logger.info(f'Deleted self-enroll course uuid:{uuid}.')
-    return redirect('self_enrollment_tool:index')
-
-
-@login_required
-@lti_role_required(const.ADMINISTRATOR)
-@lti_permission_required(settings.PERMISSION_SELF_ENROLLMENT_TOOL)
-@require_http_methods(['DELETE'])
-def unenroll_user_from_course(request, course_instance_id, user_id):
-    """
-    This functions deletes a user from a self-enroll course.
-    """
-    logger.info(
-        f'Deleting user_id:{user_id} from self-enroll course_instance_id:{course_instance_id}.')
-
-    try:
-        try:
-            course_enrollee = CourseEnrollee(
-                user_id=user_id, course_instance_id=course_instance_id)
-        except CourseEnrollee.DoesNotExist:
-            msg = f'user_id:{user_id} enrolled in course_instance_id:{course_instance_id} '
-            f'does not exists and therefore cannot be deleted.'
-            logger.warning(msg)
-            messages.warning(request, msg)
-
-        # TODO: Add code to delete or archive course enrollee record.
-        # course_enrollee.DeletedEnrollee()
-    except Exception:
-        msg = f'Unable to delete user_id:{user_id} from course_instance_id:{course_instance_id}.'
+        msg = f'Unable to delete self-enroll URL for course uuid: {uuid}.'
         logger.exception(msg)
         messages.error(request, msg)
 
-    logger.info(
-        f'Deleted user_id:{user_id} from self-enroll course_instance_id:{course_instance_id}.')
+    msg = f'Successfully deleted self-enroll URL for course (uuid: {uuid}).'
```

Comment:
This message should display a more useful  Course Metadata. Rather than uuid, course name/title that is in the first column will be useful for the user

Reply:
Updated this to not include the uuid and just be a simple successful message like we talked about 👍